src/app/admin/pages/learning-objects/learning-objects.component.html

```diff
@@ -25,6 +25,8 @@
               #scroll 
               [style.height]="'calc(100vh - ' + listViewHeightOffset + 'px)'"
               [items]="learningObjects"
+              [bufferAmount]="query.limit"
+              (vsEnd)="scroll.viewPortInfo.startIndex && getLearningObjects($event);"
```

Comment:
Probably not but I can try.

From what I understand, the vsEnd event is fired whenever the scrollbar reaches the bottom of one of the list elements. So this event fires literally every time the scrollbar passes an element. So the first condition here is to make sure that we have scrolled so that it doesn't attempt to retrieve objects in an infinite loop (since the event fires immediately when objects are loaded) and then the `getLearningObjects` function is updated to check that if it was given an event, that the scroll position just passed the last element.